packages/ra-realtime/src/buildAorAction.js

```diff
@@ -0,0 +1,11 @@
+import { FETCH_END } from 'react-admin';
+
+export default (
+    { type, payload, meta: { fetch: restType, ...meta } },
+    parsedApolloQueryResult
```

Comment:
why mention Apollo` here? Isn't this packagfe supposed to be independent from Apollo?

Reply:
Probably a left-over from when it was integrated in the simple graphql client...